src/lib/components/media/riven/item-manual-scrape.svelte

```
@@ -0,0 +1,1283 @@
+            open = false;
+            resetFlow();
+        } catch (e) {
+            console.error("Batch completion failed", e);
+            toast.error("Batch completion failed");
+        } finally {
+            loading = false;
+            batchProgress = null;
+        }
+    }
+
+	function getResolutionColor(resolution?: string): string {
```

ESLint

[error] 'getResolutionColor' is defined but never used.

(@typescript-eslint/no-unused-vars)